docs/workflows/cdn-image-proxy.md

```
@@ -0,0 +1,100 @@
+|----------|-------------|
+| `_markup/render-image.html` | `<picture>` srcset via wsrv.nl (webp+jpeg, 400/800/1200/1600w) |
+| `partials/blog/img-cropped.html` | Retina thumbnail via wsrv.nl |
+| `partials/seo/enhanced-meta-tags.html` | OG image (1200×630) via wsrv.nl |
+| `_shortcodes/img.html` | Direct image via wsrv.nl |
+
+## Solution 1: wsrv.nl (Current)
+
+Free image proxy at [wsrv.nl](https://wsrv.nl). No account needed. Supports resizing, format conversion, quality control.
+
+**URL pattern:**
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)